src/scheduler/fifo.c

```diff
@@ -2984,3 +2986,42 @@ set_validate_sched_attrs(int connector)
 
 	return 1;
 }
+
+/**
+ * @brief Validate running user.
+ * If PBS_DAEMON_SERVICE_USER is set, and user is root, change user to it.
+ *
+ * @param[in] exename - name of executable (argv[0])
+ *
+ * @retval Error code
+ * @return 0 - Failure
+ * @return 1 - Success
+ *
+ * @par Side Effects:
+ *	None
+ */
+int
+validate_running_user(char * exename) {
```

Comment:
Can this function move to more generic part of PBS? It seems to confirm daemon service user might be useful for other components too.

Reply:
I think i'd prefer to leave it here until another daemon actually uses it. Hopefully by then we'll have OOP and a daemon class to put it in.